### ISSUE TITLE
[Suggestion] Remove the kube context: user, cert & key from kubeconfig when we delete a cluster

### ISSUE THREAD

**Issue by cmoulliard** [cnoe-io/idpbuilder#458]
## Suggestion

Remove from the kubeconfig file the kube context (user, cert & key, etc) when we delete a cluster.

When we delete an idpbuilder cluster, the `kind` cluster is well removed but not the `kube context` from the kubeconfig file. This is due to the fact that we are missing to pass the path of the file to the command kind libe function `provider.Delete(name, "")`

```
// see https://github.com/cnoe-io/idpbuilder/blob/8c6cd9704ae118981a89b3c5b541db1924f31643/pkg/cmd/delete/root.go#L43

provider := cluster.NewProvider(cluster.ProviderWithLogger(kind.KindLoggerFromLogr(&logger)), detectOpt)
if err := provider.Delete(name, ""); err != nil {
		return fmt.Errorf("failed to delete cluster %s: %w", name, err)
}
```

**Comment by punkwalker** [comment]
@cmoulliard Is #362  similar to what you are expecting with kubeconfig path?

**Comment by cmoulliard** [comment]
> Is #362 similar to what you are expecting with kubeconfig path?

Yes. We will use such an issue to push a PR offering such a new `kubeconfig` path flag
